plugin/db/mysql/restore.go

```diff
@@ -94,8 +94,7 @@ func newBinlogCoordinate(binlogFileName string, pos int64) (binlogCoordinate, er
 }
 
 // SetUpForPITR sets necessary fields for MySQL PITR recovery.
-func (driver *Driver) SetUpForPITR(mysqlutilInstance mysqlutil.Instance, binlogDir string) {
-	driver.mysqlutil = mysqlutilInstance
+func (driver *Driver) SetUpForPITR(binlogDir string) {
```

Comment:
nit: can we also remove binlogDir, and remove SetUpForPITR altogether in another PR?

Reply:
I will do it in the following PR(s)